packages/viewer/src/lib/interactions/baseInteractionHandler.ts

```diff
@@ -200,6 +210,13 @@ export abstract class BaseInteractionHandler implements InteractionHandler {
     this.lastMoveEvent = undefined;
   }
 
+  protected async handleDoubleClick(event: BaseEvent): Promise<void> {
+    if (event.buttons === 4 && this.interactionApi != null) {
+      this.interactionApi.beginInteraction();
```

Comment:
This can prob be removed if with the comment above. If you use this in the future, you also need to invoke `this.interactionApi.endInteraction()` to get a final frame. Otherwise, rendering might be stuck sending back intermediate frames.